src/main/java/com/mindee/parsing/common/Job.java

```diff
@@ -21,7 +33,6 @@ public class Job {
    */
   @JsonProperty("available_at")
```

Comment:
@JsonDeserialize(using = LocalDateTimeDeserializer.class)

We should serialize available_at with the same deserializer. (I was the one who missed it)

Reply:
done, thanks